akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -91,7 +91,7 @@ private[stream] object Stages {
 
   object Fused {
     def apply(ops: immutable.Seq[Stage[_, _]]): Fused =
-      Fused(ops, name(ops.iterator.map(x ⇒ Logging.simpleName(x).toLowerCase).mkString("+")))
+      Fused(ops, name(ops.iterator.map(x ⇒ akka.event.Logging.simpleName(x).toLowerCase).mkString("+")))
```

Comment:
at some point my class was named Logging, causing this. I'll revert those once we settle for a name for it (Logging / Debug etc).